src/web/traceViewer/ui/callTab.tsx

```diff
@@ -57,10 +58,31 @@ export const CallTab: React.FunctionComponent<{
     </div>;
 };
 
-function renderValue(value: any) {
+function renderLine(metadata: CallMetadata, name: string, value: any) {
+  const { title, type } = toString(metadata, name, value);
+  let text = trimRight(title.replace(/\n/g, '↵'), 80);
+  if (type === 'string')
+    text = `"${text}"`;
+  return <div className='call-line'>{name}: <span className={type} title={title}>{text}</span></div>
+}
+
+function toString(metadata: CallMetadata, name: string, value: any): { title: string, type: string } {
+  if (metadata.method.includes('eval')) {
```

Comment:
There are also BindingCall.resolve/reject, waitForFunction and dispatchEvent

Reply:
Yes, I was thinking of generating the renderers at some point based on the protocol.